manoseimas/mps_v2/static/components/filter_component.jsx

```diff
@@ -9,6 +9,7 @@ var SortableList = React.createClass({
       sort_order: this.props.default_order,
       filter_selected: 'all',
       filter_options: null,
+      subtab_selected: this.props.active_subtab || 'lobbyists',
```

Comment:
Under what circumstances `this.props.active_subtab` is undefined or empty and we fall back to the hardcoded 'lobbyists'?

Can we instead ensure that this.props.active_subtab is always defined?